kivy/uix/widget.py

```diff
@@ -343,9 +348,28 @@ def __init__(self, **kwargs):
         if self.canvas is None:
             self.canvas = Canvas(opacity=self.opacity)
 
+        # Because we currently don't support 'on_kv_pre' in kv, we might be
+        # able to cheese this part like this:
+        #
+        # self.on_kv_pre()
+        #
+        # unless the user does this:
+        #
+        # def __init__(self, **kwargs):
+        #     super().__init__(**kwargs)
+        #     self.bind(on_kv_pre=...)
```

Comment:
This comment is not really needed because it doesn't add info on the current code, but rather is targeted for us reading the PR so please remove it.